eng/common/pipelines/templates/steps/set-daily-docs-branch-name.yml

```diff
@@ -7,7 +7,8 @@ steps:
   - pwsh: |
       $branchName = $env:DAILYDOCSBRANCHNAMEOVERRIDE
       if (!$branchName) { 
-        $branchName = "daily/$(Get-Date -Format 'yyyy-MM-dd')"
+        #$branchName = "daily/$(Get-Date -Format 'yyyy-MM-dd')"
+        $branchName = "daily/2022-03-09"
```

Comment:
One doesn't need to hard code this even for dev work. Just set the variable `DAILYDOCSBRANCHNAMEOVERRIDE` to `daily/2022-03-09` when queuing the docindex pipeline.